samples/tls/monitoring/README.md

```diff
@@ -64,20 +27,38 @@ my_repository_path/monitoring/bin/deploy_monitoring_cluster.sh
 
 ## Notes On Customization Values
 
-Set `MON_TLS_ENABLE=true` in the `$USER_DIR/monitoring/user.env` file. This variable modifies the deployment of Prometheus,
-Grafana, and Alertmanager to be TLS-enabled.
+* Set `TLS_ENABLE=true` in the `$USER_DIR/monitoring/user.env` file to specify that Prometheus, Grafana, and Alertmanager are deployed with TLS enabled for connections to these components. Due to limitations in the underlying Helm charts, some components might not have TLS enabled for in-cluster connections between components. See the **Limitations and Known Issues** section below for details.
 
-Edit `$USER_DIR/monitoring/user-values-prom-operator.yaml` and replace
+* Edit `$USER_DIR/monitoring/user-values-prom-operator.yaml` and replace
 any sample hostnames with hostnames for your deployment. Specifically, you must replace
 `host.cluster.example.com` with the name of the ingress node. Often, the ingress node is the cluster master node, but your environment might be different.
 
-## Limitations and Known Issues
+* If you are using ingress, manually populate these secrets in the `monitoring` namespace (or `MON_NS` value) **before** you deploy cluster monitoring.
+
+* kubernetes.io/tls secret - `prometheus-ingress-tls-secret`
+* kubernetes.io/tls secret - `alertmanager-ingress-tls-secret`
+* kubernetes.io/tls secret - `grafana-ingress-tls-secret`
+
```

Comment:
The notation here (and in the corresponding section in the logging readme file) is probably not clear to the reader.  The first part (e.g. kubernetes.io/tls secret) is just identifying the type of Kubernetes object to create.  The second part (alertmanager-ingress-tls-secret) identifies the name of the Kubernetes secret object within the monitoring (or logging) namespace that contains the TLS cert.    I think some sample syntax (i.e. syntax needed to create one of these secrets from a TLS cert file) would be helpful here.  @BryanEllington can you provide David with that?